lib/database/user.js

```diff
@@ -0,0 +1,5 @@
+if (Meteor.isServer) {
+	Meteor.publish('users', function() {
```

Comment:
This introduces a huge security problem.

Reply:
Yeah, i am publishing more amount of data, than it is required.

Reply:
Will fix this!